src/app/chart-docs/chart-types-docs/chart-types-main/chart-types-main.component.ts

```diff
@@ -0,0 +1,80 @@
+import { Component, OnInit, ChangeDetectionStrategy } from '@angular/core';
+import { TdMediaService } from '@covalent/core/media';
+
+@Component({
+  selector: 'app-chart-types-main',
+  templateUrl: './chart-types-main.component.html',
+  styleUrls: ['./chart-types-main.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+
+})
+export class ChartTypesMainComponent {
+
+  items: Object[] = [{
+    color: 'deep-purple-A700',
+    icon: 'insert_chart',
+    route: 'bar',
+    title: 'Bar Chart',
+  }, 
+  {
+    color: 'blue-A700',
+    icon: 'show_chart',
+    route: 'line',
+    title: 'Line Chart',
+  }, 
+  {
+    color: 'pink-A700',
+    icon: 'multiline_chart',
+    route: 'scatter',
```

Comment:
think this should be `route: 'combination',`

Reply:
@jennmedellin good catch - moving fast. I'll update it.